common/collation/ku.xml

```diff
@@ -13,32 +13,33 @@ For terms of use, see http://www.unicode.org/copyright.html
 	<collations>
 		<!-- TODO: Add collation type="search" -->
 		<collation type="standard" draft="contributed"
-				references="​http://en.wikipedia.org/wiki/Kurdish_alphabets 2013-aug-20">
-			<!-- Initial request and data see cldrbug 6527. -->
+				references="http://en.wikipedia.org/wiki/Kurdish_alphabets 2019-oct-10">
+			<!-- Initial request and data see cldrbug 6527. Subsequent data, see cldrbug 9748. -->
 			<cr><![CDATA[
 				[reorder Latn Arab]
-			
-				# Primary differences for Kurmanji letters.
+
+				# These tailorings are based on the Universal Kurdish Latin Alphabet which seeks
+				# to accomodate all three major dialects: Northern, Central, and Southern Kurdish.
```

Comment:
CLDR has data by language. This here is for ku=kmr.
Why should we add sort order rules *here* for letters used in ckb and sdh?
I might be persuaded; please try to persuade us :-)

Reply:
Thanks for clarifying that this collation is specifically for Kurmanji/Northern Kurdish. As you may know, the different varieties of Kurdish are sometimes treated as dialects and sometimes as distinct languages. I'm fine with this collation being specific to Kurmanji/Northern Kurdish, although it would probably be a good idea to explain that clearly in the comments.